### ISSUE TITLE
hardware adapters?

### ISSUE THREAD

**Issue by pjebs** [kevzettler/decompiled_dasung_drivers#1]

**Comment by kevzettler** [comment]
hi @pjebs 

> Do you know eink model they would use for that latest version that is available?

I do not know what the latest Dasungs. I'd imagine they are using the same line of e-ink models that I listed in my article.

> Also do you think it's feasible to buy a ready made unit that converts hdmi signals to the input of the e-ink module?

I am currently investigating this same question. Thanks for asking. I have in possession some of raw Eink displays but am unsure how to hook them up to anything. I believe the modules use an SPI connection, but I am not educated enough on hardware to confirm.

The modules are supposed to be compatible with this official driving board :
https://shopkits.eink.com/product/ice-driving-board/
That doesn't appear to have HDMI connection USB only.

This repository has the schematics for building a custom driving eink driving board but it looks targeted for old eink models and no hdmi support  https://github.com/julbouln/ice40_eink_controller


Do let me know if you find anything.

**Comment by pjebs** [comment]
Hi @kevzettler,

I am also doing research into it. I'm also thinking of purchasing one of the units below to investigate how they are doing it.

https://www.waveshare.com/eink-disp-103.htm?sku=17660
https://www.seeedstudio.com/10-3-inch-E-paper-Monitor-with-HDMI-Display-Interface-for-Raspberry-Pi-and-Jetson-Nano-p-4438.html 

I am formally trained in EE, but I've always been a software dev (Go) professionally so a lot has changed since my university days.

My plan is to start from first principles:

1. Accept output from HDMI decoder which should be simple to do (in theory).
2. Program a FPGA to convert the image to greyscale
3. Do what ever that board is doing to feed it into the e-ink display.

That driving board you sent a link for is for connecting to your pc to program applications to control the display. It's arguably not really for real-world use. The USB connector is for uploading new compiled C code (usually). At least it provides a clue as to what chip is needed to make the board display something.

Do you want to take this convo to email so I can explain a few other things.

**Comment by kevzettler** [comment]
@pjebs yes lets take this discussion off github. I don't want to drop my email here so please contact me on Twitter, Telegram, or Signal. Let me know if you don't have accounts on any of those and we'll work something out.

**Comment by pjebs** [comment]
planetpjs@hotmail.com

**Comment by kevzettler** [comment]
@pjebs sent
